test/pgn_test.dart

```diff
@@ -46,7 +46,7 @@ void main() {
 
   test('parse pgn roundtrip', () {
     const pgn = '1. e4 \ne5\nNf3 {foo\n  bar baz } 1-0';
-    final game = PgnGame.parsePgn(pgn, PgnGame.emptyHeaders);
```

Comment:
We should add some tests for the different error cases that can happen on `PgnGame.parsePgn` and `parseMultiGamePgn`. 
Test that it throws properly with the right error object.